porth.py

```diff
@@ -249,6 +251,16 @@ def simulate_little_endian_linux(program: Program):
                 store_addr = stack.pop()
                 mem[store_addr] = store_value & 0xFF
                 ip += 1
+            elif op.operand == Intrinsic.LOAD64:
+                addr = stack.pop()
+                byte = mem[addr]
+                stack.append(byte)
+                ip += 1
+            elif op.operand == Intrinsic.STORE64:
+                store_value = stack.pop()
+                store_addr = stack.pop()
+                mem[store_addr] = store_value & 0xFFFFFFFF
```

Comment:
`0xFFFFFFFF` is a 32 bit mask, not 64. 64 would be `0xFFFFFFFFFFFFFFFF` and I'm not even sure if it's even needed.

Reply:
Oh I seem to have miscounted. Will fix.